base/cvd/cuttlefish/host/commands/cvd/server_command/status_fetcher.cpp

```diff
@@ -43,6 +46,33 @@ struct IdAndPerInstanceName {
   unsigned id;
 };
 
+// The most important thing this function does is turn "INSTANCE_STATE_RUNNING"
+// into "Running". Some external tools (like the host orchestrator) already
+// depend on this string.
+std::string HumanFriendlyStateName(cvd::InstanceState state) {
+  std::string name = cvd::InstanceState_Name(state);
+  // Drop the enum name prefix
+  std::string prefix = "INSTANCE_STATE_";
+  if (android::base::StartsWith(name, prefix)) {
+    name = name.substr(prefix.size());
+  }
+
+  for (size_t i = 0; i < name.size(); ++i) {
+    // Replace underscores with spaces
+    if (name[i] == '_') {
+      name[i] = ' ';
+      continue;
+    }
+    // All characters but the first of each word should be lowercase
+    bool first = (i == 0 || name[i-1] == ' ');
+    if (!first) {
+      name[i] = std::tolower(name[i]);
```

Comment:
Because `std::string` is made of `char`s, note that for `std::tolower`, [cppreference writes](https://en.cppreference.com/w/cpp/string/byte/tolower)

> Like all other functions from [`<cctype>`](https://en.cppreference.com/w/cpp/header/cctype), the behavior of `std::tolower` is undefined if the argument's value is neither representable as `unsigned char` nor equal to [`EOF`](https://en.cppreference.com/w/cpp/io/c). To use these functions safely with plain `char`s (or `signed char`s), the argument should first be converted to `unsigned char`:
```c++
static_cast<char>(std::tolower(static_cast<unsigned char>(ch)))
```